ui-v2/app/templates/dc/services/-addresses.hbs

```diff
@@ -0,0 +1,25 @@
+{{#if item.TaggedAddresses }}
+{{#tabular-collection
+    data-test-addresses
+    items=(object-entries item.TaggedAddresses) as |taggedAddress index|
```

Comment:
As a minimal context reviewer my only question was how `TaggedAddreses` are populated. I assume this is just rendering what is provided by the API in the same object.

Reply:
Yup, good assumption! That's part of the response JSON from the API, `TaggedAddresses` on `Service` are a new thing added by the gateway support.

Any capitalised property you see in the templates is an API property, that anyone can look up in the docs if they ever need to.